app/frontend/src/features/profile/view/leaveReference/formSteps/Text.tsx

```diff
@@ -53,24 +54,34 @@ export default function Text({
 
   return (
     <form className={classes.form} onSubmit={onSubmit}>
-      <ReferenceStepHeader name={user.name} referenceType={referenceType} />
+      <ReferenceStepHeader
+        name={user.name}
+        referenceType={referenceType}
+        isSubmitStep={false}
+      />
       <TextBody className={classes.text}>{TEXT_EXPLANATION}</TextBody>
       <TextBody className={classes.text}>{PUBLIC_ANSWER}</TextBody>
-      {errors && errors.text?.message && (
+      {errors.text?.message && (
         <Alert className={classes.alert} severity="error">
           {errors.text.message}
         </Alert>
       )}
+      {errors.text?.type === "required" && (
+        <Alert className={classes.alert} severity="error">
+          {REQUIRED}
```

Comment:
As above, you won't need this block if you do `required: REQUIRED` instead of true